code/controllers/subsystem/persistence.dm

```diff
@@ -294,6 +295,42 @@ SUBSYSTEM_DEF(persistence)
 	for(var/obj/structure/sign/painting/P in painting_frames)
 		P.load_persistent()
 
+	json_file = file("data/pending_art_metacoin.json")
+	if(fexists(json_file))
+		pending_art_metacoin = json_decode(file2text(json_file))
+
+	if(paintings && length(paintings["library"]) && !paintings["library"][1]["price"]) //this should only ever happen once - adding owner and price values to legacy paintings
```

Comment:
I feel like this should be able to detect legacy paintings always, in the case that testmerge is applied / removed or this is reverted. Perhaps check the last painting ratehr than the first one?